collection-scripts/gather

```diff
@@ -49,3 +49,9 @@ done
 
 # Gather Windows Kubernetes component logs
 /usr/bin/gather_windows_node_logs
+
```

Comment:
can we also gather core dumps for ovn-controller, northd?

Reply:
also can we get RPM versions of OVN

Reply:
Yep should be able to handle that! Just need to figure out the necessary core dump procedure + commands